src/main/kotlin/no/nav/familie/ef/sak/repository/VedtaksbrevRepository.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.sak.repository
+
+import no.nav.familie.ef.sak.repository.domain.Vedtaksbrev
+import org.springframework.stereotype.Repository
+import java.util.*
+
+@Repository
+interface VedtaksbrevRepository : RepositoryInterface<Vedtaksbrev, UUID>, InsertUpdateRepository<Vedtaksbrev> {
+
+    fun findByBehandlingId(behandlingId: UUID): List<Vedtaksbrev>
```

Comment:
Burde vi heller ha `findByBehandlingId(....): Vedtaksbrev`  - slik at vi forventer ett eller ingen innslag?

Reply:
I datamodellen vi har lagt opp til kan en behandling ha to vedtaksbrev. Et før og et etter totrinns. Men det er kanskje bedre å bare ha en entitet pr. behandling, og så heller oppdatere det til å reflektere riktig "tilstand"?